index.src.html

```diff
@@ -5,6 +5,7 @@ <h1>Self-Review Questionnaire: Security and Privacy</h1>
 Shortname: security-privacy-questionnaire
 Level: 1
 Editor: Lukasz Olejnik, Independent researcher, me@lukaszolejnik.com
+Editor: Jason Novak, Apple Inc., https://apple.com
```

Comment:
Erm, do you have a mail or a site? Otherwise, should I edit out my mail to my site? Asking on consistency.

Reply:
I've seen folks put either a site or an email.... I went with site.